arclines/plots.py

```diff
@@ -75,19 +79,26 @@ def chk_line_list(wave):
 
     # Unknowns
     U_lines = src_dict['U_lines']
-    clrs = ['red', 'orange']  # Used, not used
+    clrs = ['red', 'orange', 'green']  # Used, not used, ??
```

Comment:
Two comments - one about color-blindedness (red/green) and the other about the color correspondence (which seems counter intuitive, to me at least). Should we have the following:
used = blue
not used = red
?? = yellow/black

Reply:
I have to admit my own color scheming confuses me here.

Am going to keep it for now but you can bet it will change again..